robottelo/config/base.py

```diff
@@ -879,7 +879,7 @@ def _validate_robottelo_settings(self):
         """Validate Robottelo's general settings."""
         validation_errors = []
         browsers = ('selenium', 'docker', 'saucelabs')
-        webdrivers = ('chrome', 'firefox', 'ie', 'phantomjs', 'remote')
+        webdrivers = ('chrome', 'edge', 'firefox', 'ie', 'phantomjs', 'remote')
```

Comment:
It seems as pretty redundant statement to keep `ie` and `edge` in the same context as it is the same platform which cannot exist on the same machine simultaneously.